### DISCUSSION TITLE
How to use alipay for my bill?

### DISCUSSION THREAD

**Discussion by linjiahao962889027** [community/community#19116] [Other Feature Feedback, Questions, & Ideas] [answered]
I want to use Alipay to pay my GitHub copilot bill, but it doesn't seem to have this option. This is a very common payment method in Chinese Mainland. If not, I might choose to uninstall GitHub copilot.

**Comment by melvincwng** [comment, marked as answer]
Hi @linjiahao962889027, based on GitHub documentation over [here](https://docs.github.com/en/billing/managing-your-github-billing-settings/adding-or-editing-a-payment-method), it seems there are only two modes of payment on GitHub - either a) Credit Card or b) PayPal account.

Doesn’t seem like Alipay is one of the payment methods available.

**Reply by melvincwng** [reply]
@linjiahao962889027, hope this answer clarifies.

Thanks and have a nice day 😊

**Reply by melvincwng** [reply]
@linjiahao962889027, also I just want to add on, Github Copilot is free to use for verified students.

So, if you are a current student studying at a higher education institute, you might want to take note of this free offer to use Github Copilot.

Hope this helps :)

**Reply by linjiahao962889027** [reply]
> It's a pity that I just graduated last year. I think GitHub copilot has provided a lot of help in my work, and I am willing to pay for it. But my credit card doesn't seem to support us dollars. Alipay can solve my problem. Moreover, few people use PayPal in China. Of course, if you support wechat payment, I think it's OK. I hope you can consider Chinese users. Thank you!

**Reply by melvincwng** [reply]
@linjiahao962889027, unfortunately there doesn't seem to be any workaround solutions other than the usage of credit card or PayPal. 😞 

Kindly mark this response as 'Mark as answer' so that we can let others know this issue is closed/addressed.

Thanks
